### ISSUE TITLE
508 Checkup Review

### ISSUE THREAD

**Issue by janette** [GetDKAN/dkan#1088]
## Description

 As new features are added we need to review 508 compliance to make sure we have coverage as well as items missed on prior checks.
## Tasks

**Dataset page**
- missing alt text on map tile images, technically a red flag but in reality a bit absurd

![screenshot_4_25_16__1_23_pm](https://cloud.githubusercontent.com/assets/314172/14794546/4db8abb2-0aea-11e6-8459-8022418165c7.png)

**Resource: Recline preview: Map**
-  missing labels on latitude and longitude fields
-  missing alt text on map tiles
-  missing alt text on mapquest logo/icon

![screenshot_4_25_16__1_13_pm](https://cloud.githubusercontent.com/assets/314172/14794501/0e47f320-0aea-11e6-8aaa-b4849d05fae3.png)

**Resource:Recline preview: Graph**
-  missing labels on form fields

![screenshot_4_25_16__1_24_pm](https://cloud.githubusercontent.com/assets/314172/14794563/68e47538-0aea-11e6-905d-daf8baba4511.png)
## Acceptance Criteria

I should not see red flags when viewing the following pages with the WAVE filter.
- [ ] front page
- [ ] resource page: grid, graph, and map previews
## Test Updates

Ask devops team if tests possible for 508 issues

**Comment by dafeder** [comment]
@janette can you do some research to see if there are any 508 tools out there that we could plug into our CI? They would need an API that returns machine-readable results. I would start here:

https://www.w3.org/WAI/ER/tools/

**Comment by janette** [comment]
http://pa11y.org/
https://github.com/paypal/AATT

**Comment by janette** [comment]
Fixed!
https://github.com/NuCivic/dkan/pull/1658
https://github.com/NuCivic/dkan/pull/1631
https://github.com/NuCivic/dkan/pull/1626
https://github.com/NuCivic/dkan/pull/1602
https://github.com/NuCivic/dkan/pull/1575
https://github.com/NuCivic/dkan/pull/1541

**Comment by janette** [comment]
Whoops one more issue after selecting filters: 
![dkan](https://user-images.githubusercontent.com/314172/28220120-eb7d878a-6883-11e7-8fc1-39fb9504bf61.png)

**Comment by kimwdavidson** [comment]
Is this last item still an issue?

**Comment by janette** [comment]
@kimwdavidson yes

**Comment by susisiman** [comment]
Recline will not be used in future releases of DKAN. Closed due to inactivity. Feel free to open!
